### PR TITLE
cat: minor cleanups

### PR DESCRIPTION
**Pull request by miallo**
In our code review group we found a few minor simplifications :)

### CODE REVIEW

**Comment by sylvestre** [comment]
what is a review group ? :)
please run rustfmt on it

**Comment by miallo** [comment]
> what is a review group ? :)

Together with a few people we meet once a week to review code. This is our first project to look into :)

> please run rustfmt on it

Sorry :see_no_evil: - of course I will do that

**Comment by miallo** [comment]
@sylvestre I am not sure how it is usually handled with force-pushing vs. append-only in this project and I couldn't find anything in [CONTRIBUTING.md](https://github.com/uutils/coreutils/blob/main/CONTRIBUTING.md). I hope with this small patch-set it is relatively easy to look at anyways...

File: src/uu/cat/src/cat.rs [hunk 1]
```diff
@@ -325,15 +325,15 @@ fn cat_path(
...
-        return cat_handle(&mut handle, options, state);
-    }
     match get_input_type(path)? {
+        InputType::StdIn => {
+            let stdin = io::stdin();
+            let mut handle = InputHandle {
+                reader: stdin,
+                is_interactive: atty::is(atty::Stream::Stdin),
+            };
+            return cat_handle(&mut handle, options, state);
```
Comment (sylvestre) [inline src/uu/cat/src/cat.rs:335]:
```suggestion
            cat_handle(&mut handle, options, state)
```

**Comment by tertsdiepraam** [comment]
Both are fine. If you force push we can merge it directly and if you add commits we'll probably squash and merge.

File: src/uu/cat/src/cat.rs (hunk 1 above)
Comment (miallo) [reply src/uu/cat/src/cat.rs]: That semicolon was too much in my first try to resolve it... :D

**Comment by tertsdiepraam** [comment]
Thank you!

**Comment by miallo** [comment]
Thank all of you for maintaining this great project! :tada:

**Pull request merged**